### ISSUE TITLE
button padding in chrome

### ISSUE THREAD

**Issue by jscharett** [US-CBP/cbp-theme#188]
1.8.1

The following style causes buttons to be askew in chrome

```
.btn, .switch-toggle a, .ui-datepicker button.ui-state-default {
    font-weight: 400;
    font-size: 13px;
    margin: 1px 0;
}
```

![image](https://user-images.githubusercontent.com/7661580/34309157-98fe2f36-e71e-11e7-9a5b-b1ad7df77151.png)

**Comment by r1teshkumar** [comment]
@jscharett , what is the HTML markup that you have used here.

**Comment by jscharett** [comment]
see #187

**Comment by davezen1** [comment]
Resolved by PR #194
